pom.xml

```diff
@@ -258,6 +258,13 @@
         </dependency>
     </dependencies>
 
+    <pluginRepositories>
```

Comment:
This should not be being added, we should either mirror via Nexus, or not be needing them. In this case again, Snapshots = Bad